src/InfluxDB2/QueryApi.php

```diff
@@ -26,13 +31,14 @@ public function __construct(array $options)
     }
 
     /**
+     * Executes the Flux query and returns the unparsed raw result
      *
-     * @param string $query query the flux query to execute. The data could be represent by string, Query
-     * @param null $org specifies the source organization
-     * @param null $dialect csv dialect
+     * @param string|Query $query flux query to execute. The data could be represent by string, Query
+     * @param string|null $org specifies the source organization
+     * @param Dialect|null $dialect csv dialect
      * @return string
      */
-    public function queryRaw(string $query, $org = null, $dialect = null): ?string
+    public function queryRaw($query, string $org = null, ?Dialect $dialect = null): ?string
```

Comment:
Please be consistent. Mark all nullable types with `?` or remove `?` from `?Dialect`.

```suggestion
    public function queryRaw($query, ?string $org = null, ?Dialect $dialect = null): ?string
```